<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CStatusMessageForm</class>
 <widget class="QFrame" name="CStatusMessageForm">
  <property name="windowTitle">
   <string>Status message</string>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Sunken</enum>
  </property>
  <layout class="QGridLayout" name="gl_StatusMessageForm">
   <property name="leftMargin">
    <number>4</number>
   </property>
   <property name="topMargin">
    <number>4</number>
   </property>
   <property name="rightMargin">
    <number>4</number>
   </property>
   <property name="bottomMargin">
    <number>4</number>
   </property>
   <property name="spacing">
    <number>4</number>
   </property>
   <item row="2" column="3" colspan="6">
    <widget class="QLineEdit" name="le_Categories">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="2">
    <widget class="QLabel" name="lbl_Categories">
     <property name="text">
      <string>Categories:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="6">
    <widget class="QLabel" name="lbl_Timestamp">
     <property name="text">
      <string>Timestamp:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="lbl_Severity">
     <property name="text">
      <string>Severity:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1" colspan="8">
    <widget class="QPlainTextEdit" name="te_Message">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="5">
    <widget class="QLabel" name="lbl_SeverityIcon">
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="../../blackmisc/blackmisc.qrc">:/diagona/icons/diagona/icons/question.png</pixmap>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QLineEdit" name="le_Severity">
     <property name="maximumSize">
      <size>
       <width>200</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="4">
    <spacer name="hs_1">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="7">
    <widget class="QLineEdit" name="le_Timestamp">
     <property name="maximumSize">
      <size>
       <width>200</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="8">
    <spacer name="hs_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../blackmisc/blackmisc.qrc"/>
 </resources>
 <connections/>
</ui>
